modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -17,6 +24,16 @@ A loader for loading an [Indexed 3d Scene (I3S) layer](https://github.com/Esri/i
 | Data Format    | [Data formats](#data-formats)                       |
 | Supported APIs | `load`, `parse`                                     |
 
+## I3S format versions support
+- `I3S 1.6`
```

Comment:
This list can be a bit unclear. We are listing things we implemented but I don't think it will be completely clear to a typical reader if that covers everything.  
- E.g. can we claim full support for 1.6-1.8, if so this more a description of the I3S format evolution rather than the `I3SLoader`. As it stands these look like a list of highlights.
- Or can we list something as still unsupported? Is it worth calling out things like region support in attributes and other things you worked on?

Should we also make comments on SLPK support? (currently not supported by I3SLoader but partially supported by tile converter)

Reply:
@ibgreen agreed. Some features such as what we support in DXT are more for completeness (for example, i3s standard also supports just DXT1 & DXT5 as well). We'll have similar issues with basis (we really heavily on ETC1S compression (less so on UASTC (though its supported).

I think its a great idea to have an MD table with yes/no with links to i3s spec.

There are some features missing that are critical and will be good to support (for example lack of region support).

@belom88 can we get a list goin on and I can help also edit in relation to the i3s spec it implements/or misses.

Reply:
Yes, it would be cool to work towards 100% support for the I3S standard, that would be a very powerful statement to users (and I think we are getting pretty close).

That would require point cloud support, improved SPLK support and a few other smaller things. Having a table here showing what we are missing should be helpful to users and a good motivation to us.

At some point we may also want to add a performance section, and more information on how visualization features are supported in deck.gl (probably a separate page).